dominion-pax-exam-driver/src/main/java/org/codice/dominion/pax/exam/options/karaf/extensions/InstallFeatureExtension.java

```diff
@@ -71,8 +71,10 @@
     }
     long timeout = annotation.timeout();
 
-    if (timeout == -1L) { // default to 90s per features
-      timeout = TimeUnit.SECONDS.toMillis(90) * names.length;
+    if (timeout != -1L) {
+      timeout = annotation.units().toMillis(timeout);
+    } else { // default to 3 minutes per features
+      timeout = TimeUnit.MINUTES.toMillis(3L) * names.length;
```

Comment:
How many features are there generally? If all features timeout, would this make a test take 3 minutes * numberOfFeatures in length?